### PR TITLE
Simplify error reporting

### PR DESCRIPTION
**Pull request by mkondratek**
I added a mechanism to automatically derive the about info. A user no longer needs to find and copy paste it.
I simplified the bug report template. Some users showed frustration about the reporting system - too many required fields that are not filled eventually. A single required field remains.

## Test plan
1. Do the report 😃

### CODE REVIEW

File: src/main/kotlin/com/sourcegraph/cody/statusbar/ReportCodyBugAction.kt [hunk 1]
```diff
@@ -2,10 +2,11 @@ package com.sourcegraph.cody.statusbar
...
 import com.intellij.ide.BrowserUtil
 import com.intellij.openapi.actionSystem.AnActionEvent
+import com.sourcegraph.cody.error.CodyErrorSubmitter
 import com.sourcegraph.common.ui.DumbAwareEDTAction
 
 class ReportCodyBugAction : DumbAwareEDTAction("Open GitHub To Report Cody Issue") {
-  override fun actionPerformed(p0: AnActionEvent) {
-    BrowserUtil.open("https://github.com/sourcegraph/jetbrains/issues/new?template=bug_report.yml")
+  override fun actionPerformed(event: AnActionEvent) {
+    BrowserUtil.open(CodyErrorSubmitter().getEncodedUrl(event.project))
```
Comment (mkondratek) [inline src/main/kotlin/com/sourcegraph/cody/statusbar/ReportCodyBugAction.kt:10]: let's derive the url the same way as everywhere else

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -6,7 +6,6 @@ description: "File a bug report for any of the JetBrains IDEs: IntelliJ, Goland,
 title: "bug: "
 labels:
   - bug
-  - team/jetbrains
```
Comment (mkondratek) [inline .github/ISSUE_TEMPLATE/bug_report.yml:9]: redundant

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -16,28 +15,17 @@ body:
         Thanks for taking the time to fill out this bug report! Please include as much information as possible to help us fix the issue. 
 
         Tip: You can attach images or videos by dragging it into the text box.
-  - type: input
-    attributes:
-      label: Cody Version
```
Comment (mkondratek) [inline .github/ISSUE_TEMPLATE/bug_report.yml:21]: The Cody version will be included in IDE Information

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -16,28 +15,17 @@ body:
...
       required: true
   - type: textarea
     attributes:
-      label: Describe the bug
-      description: Description of what the bug is. Please include steps on how to reproduce the behaviour.
-    validations:
-      required: true
-  - type: textarea
-    attributes:
-      label: Expected behavior
```
Comment (mkondratek) [inline .github/ISSUE_TEMPLATE/bug_report.yml:39]: too complicated. we received a lot of error with no details filled in anyway. a single required field should be enough

File: src/main/kotlin/com/sourcegraph/cody/error/CodyErrorSubmitter.kt [hunk 5]
```diff
@@ -49,14 +58,12 @@ class CodyErrorSubmitter : ErrorReportSubmitter() {
     return "bug: $title"
   }
 
-  private fun getVersion() =
-      formatAttributes(
-          "Plugin version" to pluginDescriptor?.version,
-          "IDE version" to ApplicationInfo.getInstance().build.toString())
+  private fun getAboutText(project: Project?) = AboutDialog(project).extendedAboutText
```
Comment (mkondratek) [inline src/main/kotlin/com/sourcegraph/cody/error/CodyErrorSubmitter.kt:61]: 🚀

**Review by dominiccooney — APPROVED** [review]
Splendid!

**Pull request merged**